src/components/organisms/AuthenticationModal/AuthenticationModal.tsx

```diff
@@ -2,38 +2,70 @@ import React, { useState } from "react";
 import { Modal } from "react-bootstrap";
 import LoginForm from "./LoginForm";
 import RegisterForm from "./RegisterForm";
+import SaveCardComponent from "./SaveCardForm";
 import "./AuthenticationModal.scss";
 
 interface PropsType {
   show: boolean;
   onHide: () => void;
   afterUserIsLoggedIn?: () => void;
+  showLogin?: boolean;
 }
 
 const AuthenticationModal: React.FunctionComponent<PropsType> = ({
   show,
   onHide,
   afterUserIsLoggedIn,
+  showLogin,
 }) => {
-  const [showRegisterForm, setShowRegisterForm] = useState(false);
-  const displayLoginForm = () => setShowRegisterForm(false);
-  const displayRegisterForm = () => setShowRegisterForm(true);
+  const [showRegisterForm, setShowRegisterForm] = useState(!showLogin);
+  const [showLoginForm, setShowLoginForm] = useState(showLogin);
+  const [showSaveCardForm, setShowSaveCardForm] = useState(false);
+
+  const displayLoginForm = () => {
+    setShowRegisterForm(false);
```

Comment:
Why do we need the 2 separate variables? They seem to be 2 states of the same thing (always display one or the other).